service/initialise.go

```diff
@@ -49,6 +65,21 @@ func (e *Init) DoGetHTTPServer(bindAddr string, router http.Handler) HTTPServer
 	return s
 }
 
+// DoGetMongoDB returns a MongoDB
+func (e *Init) DoGetMongoDB(ctx context.Context, cfg *config.Config) (MongoServer, error) {
+	mongodb := &mongo.Mongo{
+		Collection: cfg.MongoConfig.Collection,
+		Database:   cfg.MongoConfig.Database,
+		URI:        cfg.MongoConfig.BindAddr,
```

Comment:
The above values in the struct only appear to be used in the init function. Consider refactoring to make them parameters of the init function rather than members of the struct.